examples/token/contracts/Connected.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@zetachain/protocol-contracts/contracts/evm/GatewayEVM.sol";
+import {RevertContext} from "@zetachain/protocol-contracts/contracts/Revert.sol";
+
+contract Connected is ERC20, Ownable {
```

Comment:
maybe we should start to use 2stepOwner instead of ownable. now it's officially part of OZ and not just an extra

Reply:
https://github.com/zeta-chain/example-contracts/pull/209/commits/3bbcab0bd0909b44fb9a0f411a60c732130270b0